generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
```

Comment:
nit: I really prefer not renaming packages, and opt for renaming vars where possible. Keep it as `path` and rename `path` var below to `p` ([p]ath), `fp` ([f]ile[p]ath), or `pth` (p[a]th)